classes/migration/install/CommonMigration.inc.php

```diff
@@ -202,7 +202,7 @@ public function up(): void
         // Templates for emails.
         Schema::create('email_templates', function (Blueprint $table) {
             $table->bigInteger('email_id')->autoIncrement();
-            $table->string('email_key', 64)->comment('Unique identifier for this email.');
+            $table->string('email_key', 255)->comment('Unique identifier for this email.');
```

Comment:
@asmecher you ok with these changes?

Reply:
@asmecher just a quick reminder on this: ok to extend the length of the `email_key` column here?